src/types.js

```diff
@@ -234,9 +234,27 @@ export interface IBaseDatabase {
   remove(collectionName: string, query: Object): Promise<*>;
 }
 
+export interface ILoggerCreate {
+  static createLogger(aName: string): ILogger;
+  static createModuleLogger(aModule: any): ILogger;
+}
 export interface ILogger {
-  static error(params: Array<any>): void;
-  static info(params: Array<any>): void;
-  static log(params: Array<any>): void;
-  static warn(params: Array<any>): void;
+  debug (): boolean;
+  debug (line: string): void;
+  debug (additional: ?Object, line: string): void;
+  error (): boolean;
+  error (line: string): void;
+  error (additional: ?Object, line: string): void;
+  fatal (): boolean;
+  fatal (line: string): void;
+  fatal (additional: ?Object, line: string): void;
+  info (): boolean;
+  info (line: string): void;
+  info (additional: ?Object, line: string): void;
+  warn (): boolean;
```

Comment:
are you using something else then "npm run lint" from package.json?  I  check my PR using this command